dev.mrquantumoff.mcmodpackmanager.yml

```diff
@@ -0,0 +1,32 @@
+# Flatpak Manifest
+app-id: dev.mrquantumoff.mcmodpackmanager
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk  
+command: minecraft-modpack-manager
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+
+modules:
+  - name: mcmodpackmanager
+    buildsystem: simple
+    build-commands:
+      - ar -vox minecraft-modpack-manager_*_amd64.deb
+      - tar -xpvf data.tar.gz
+      - install -Dm755 -t /app/bin usr/bin/minecraft-modpack-manager
+      - install -D dev.mrquantumoff.mcmodpackmanager.metainfo.xml /app/share/metainfo/dev.mrquantumoff.mcmodpackmanager.metainfo.xml
+      - sed -i 's/Icon=minecraft-modpack-manager/Icon=dev.mrquantumoff.mcmodpackmanager/g' usr/share/applications/minecraft-modpack-manager.desktop
+      - install -D usr/share/applications/minecraft-modpack-manager.desktop  /app/share/applications/dev.mrquantumoff.mcmodpackmanager.desktop
+      - install -D usr/share/icons/hicolor/256x256/apps/com.mrquantumoff.mcmodpackmanager.png  /app/share/icons/hicolor/256x256/apps/dev.mrquantumoff.mcmodpackmanager.png 
+    sources:
+      - type: file
+        path: dev.mrquantumoff.mcmodpackmanager.metainfo.xml
```

Comment:
this file is still not managed upstreeam? I should be.

Reply:
It will be resolved with 5.0.1 release

Reply:
it is resolved now.